src/random_test.cpp

```diff
@@ -1608,28 +1609,92 @@ void Table::DeleteRandomRow(Thd1 *thd) {
     }
   }
 
-  std::string sql = "DELETE FROM " + name_ + " WHERE " +
-                    columns_->at(where)->name_ + "=" +
+  std::string sql1 = "DELETE FROM " + name_ + " WHERE " +
+                    columns_->at(where)->name_ + " = " +
                     columns_->at(where)->rand_value();
+  std::string sql2 = "DELETE FROM " + name_ + " WHERE " +
+                    columns_->at(where)->name_ + " >= " +
+                    columns_->at(where)->rand_value() + " AND " +
+                    columns_->at(where)->name_ + " <= " +
+                    columns_->at(where)->rand_value();
+  std::string sql3 = "DELETE FROM " + name_ + " WHERE " +
+                    columns_->at(where)->name_ + " IN (" +
+                    columns_->at(where)->rand_value() + "," +
+                    columns_->at(where)->rand_value() + ")";
+  std::string sql4 = "DELETE FROM " + name_ + " WHERE " +
+                    columns_->at(where)->name_ + " BETWEEN " +
+                    columns_->at(where)->rand_value() + " AND " +
+                    columns_->at(where)->rand_value();
+  std::string sql5 = "DELETE FROM " + name_ + " WHERE " +
+                    columns_->at(where)->name_ + " LIKE '%" +
```

Comment:
how does **like**  will work with int, float, double.

Reply:
Just the usual string way.
for ex 
mysql> select * from mohit where b like '%67';
+------+--------+-----------+
| a    | b      | c         |
+------+--------+-----------+
| 1001 | 23.567 | allahabad |
+------+--------+-----------+
1 row in set (0.00 sec)

Reply:
then it could delete hugs records. 
would suggest using very little probability for such sql.

Reply:
Ok